docs/community/com-query/grafana-connector.md

```diff
@@ -51,26 +58,26 @@ From `grafana-7-x.x/` run the following command:
 ### Add Datasource
 
 - Sign in or create credentials (you can use admin/admin if logging in for the first time)
-- Add Molecula as a datasource in Grafana and enter FeatureBase server information
+- Add Featurebase as a datasource in Grafana and enter FeatureBase server information
     - The welcome page should include a link to the [Add data source page](http://localhost:3000/datasources).
       - See the [Grafana docs](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source) for more details.
-    - If your molecula-datasource plugin is installed, it should appear in the data source list.
-
+    - If your featurebase-datasource plugin is installed, it should appear in the data source list.
 
 ![Grafana Welcome](/assets/images/grafana-welcome.png "Grafana Welcome")  
```

Comment:
It is a bad idea to rely on screenshots to communicate required information:
* they won't be accessible to all users
* they need to be kept up to date
It's better to write out the specifics in plain text which is faster to update.

Reply:
@rachithrr you can skip this